### DISCUSSION TITLE
Pypsa local copy and fork updating procedure on terminal

### DISCUSSION THREAD

**Discussion by Emre-Yorat89** [pypsa-meets-earth/pypsa-earth#515] [General]
Hi,

I have updated my project folder and fork from terminal and if you want to update this way your project I attached a pdf file that depicts the procedure of what commands I have used also shared what errors I have taken and what steps I have followed to solve the errors.

[terminal_project_updating.pdf](https://github.com/pypsa-meets-earth/pypsa-earth/files/10009452/terminal_project_updating.pdf)

**Comment by ekatef** [comment]
@Emre-Yorat89, fantastic picture! Thanks a lot.
@max, @AnasAlgarei, can we probably add this diagram to the Frequently Asked Questions section, what do you think?

**Comment by davide-f** [comment]
Thanks @Emre-Yorat89 ! Much appreciated

@ekatef I agree that it is for the FAQ.

The image, refined, could go into the documentation package in the folder documentation/doc/img.
Emre, would you like to do a PR and upload it there?
Before uploading, please revise the pdf:
- check the rectangle that is supposed to be red top right, to my understanding
- Adjust text and and arrow size

Katia and Anas are working on the documentation of pypsa-earth and they could reference your pdf.

**Comment by pz-max** [comment]
@ekatef & @davide-f 
For the FAQ we can just link this Github discussion or use the link of the pdf: 
https://github.com/pypsa-meets-earth/pypsa-earth/files/10009452/terminal_project_updating.pdf
Probably no need to dump it in `documentation/doc/img`.

**Comment by AnasAlgarei** [comment]
Nice work Emre! I agree that it can be part of the FAQ, we can refer to it in the Hackathon's github_exercise maybe?

**Comment by Emre-Yorat89** [comment]
I have refined the pdf. If more adjustments are necessary I can change it to your recommendations.

[terminal_project_updating2.pdf](https://github.com/pypsa-meets-earth/pypsa-earth/files/10020690/terminal_project_updating2.pdf)

**Reply by pz-max** [reply]
Hi @Emre-Yorat89 , sorry for the late response. I think it looks good.
(Just some tips in case it's not noticeable. People often like to align text and symbols across documents & use similar sizes e.g. for arrows. I know this is not a beauty contest & you don't need to work on this :) )

@ekatef now we only need the FAQ section :)
